### DISCUSSION TITLE
Fluxstore Delivery Boys [done] ✅

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5071] [Working on Features]
Support delivery boys feature on the Fluxstore Multi-Vendor and Manager app
⚡️FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by HarrySmith01** [comment]
I am not sure what exactly it means "Support delivery boys feature on the Fluxstore Multi-Vendor and Manager app" but what we are looking for is a Delivery App for delivery boys 

Below are example:

https://codecanyon.net/item/delivery-person-for-food-grocery-pharmacy-stores-react-native-wordpress-woocommerce-app/27293549

To  understand it features and how it works, I would suggest you to check above app.

More example:

https://codecanyon.net/item/delivery-boy-for-groceries-foods-pharmacies-stores-flutter-app/26615174

**Reply by henryq30** [reply]
I want the same app for delivery boy, let me know when this is released.

**Reply by HarrySmith01** [reply]
To get the app you see in above link then you need to vote up for that.

**Reply by sam-shl** [reply]
Better to be supported and integrated with applicable vendors

**Reply by Haniel5** [reply]
@HarrySmith01 @henryq30 @sam-shl Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by dknaoremcha** [comment]
I want this feature, without this it's hard to manage delivery services according to the orders.

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by sam-shl** [comment]
This feature is very essential

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by sirlojik** [comment]
This feature is very integral to the businesses we are running. I guess that is why wcfm added it. I have asked in support forums... Remember as a provider of this feature to my clients, they will ask me about delivery feature since wcfm already provides that feature. I just purchased the store manager app and i cannot see assigning a delivery agent. This should have been in fluxstore manager 20 months ago. :-)

**Reply by sirlojik** [reply]
Guys, i want to launch my platform but am not sure if i should wait for this feature. what is the update cycle for fluxstore store manager? I wouldnt even mind using a freelancer, this is the only feature remaining. I guess, i will have to ask my vendors to use wcfm store manager for this feature. and use a delivery boy app on codecanyon

**Reply by minhcasi** [reply]
@sirlojik - this feature is one of the top features that need to be released, please stay tuned with our update 👍

**Comment by amokour** [comment]
delivery boy app is needed to be linked directly with fluxstore user app. that will rocket your product.

**Reply by sirlojik** [reply]
They need to add assigning of delivery boys. The delivery app will work after that.

**Reply by Haniel5** [reply]
@sirlojik Thank you for your suggestion. We are developing the delivery boy assign feature from FluxStore Manager https://1.envato.market/Kkdxz - It will be available on the new version of FluxStore Manager. Best regards and stay safe!

**Comment by sirlojik** [comment]
They know this. The question is how long before the update?

**Reply by amokour** [reply]
I know that they know.. I am showing interest only :)

**Reply by sirlojik** [reply]
i understand, me too

**Reply by Haniel5** [reply]
@sirlojik @amokour Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by zanyara** [comment]
I also would like to see this feature. Then you could have three decent apps, one for customer, one for vendor, and one for delivery persons. 
If you want to make it even better, if the delivery person has multiple items to deliver (which is very likely) your app could find the best rout on the map to deliver the packages one by one. 
That is how the Amazon delivery app works. I used to do delivery for Amazon. Amazon will try to bundle about 40 orders together which all have delivery address close to each other (within 5 km radius) then the driver gets all the orders and the app tells you where to to deliver each order one by one. It was very efficient and made the delivery process really easy. You could benefit from that model if you like.

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by minhcasi** [comment]
Hi friends, just sharing that our team is working on this feature and put heavy development effort to release it in the next couple of weeks 👍

**Reply by saber93** [reply]
Great news

**Reply by tworindia** [reply]
Great, release ASAP, pls integrate with WCFM Delivery driver also, then your three apps will rocks.

**Reply by saber93** [reply]
Hey Bro,

I'm really still waiting for that release!!!
any update?

**Reply by Haniel5** [reply]
@saber93 @tworindia Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by iamshivansh59** [comment]
Everything is awesome but only delivery boy app is missing..

Release it soon as possible..

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by CeoAlexMartinez** [comment]
Esperando esta característica...

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by anshverma14** [comment]
Great Idea, will be useful

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!

**Comment by abedfatafta** [comment]
if you enable it in multivendor keep both ( store shipping and admin shipping )

**Reply by Haniel5** [reply]
Hi, FluxStore Delivery Boy is released https://1.envato.market/4edvXL ⚡️$39 - Discount 50% $19⚡️ - Thank you!
